core/iwasm/fast-jit/jit_dump.c

```diff
@@ -15,45 +15,45 @@ jit_dump_reg(JitCompContext *cc, JitReg reg)
 
     switch (kind) {
         case JIT_REG_KIND_VOID:
-            os_printf("VOID");
+            LOG_VERBOSE("VOID");
```

Comment:
using LOG_VERBOSE won't dump the log when log level is smaller than 5. And for fast-jit dump log, the log is has format and is complex and long, I am afraid adding timestamp and thread id may make it a little noisy:
```
; Function: $f87
; Constant table sizes: VOID=0 I32=0 I64=1 F32=0 F64=0 V64=0 V128=0 V256=0
; Label number: 4
; Instruction number: 43
; Register numbers: VOID=0 I32=16 I64=40 F32=16 F64=16 V64=0 V128=0 V256=0
; Label annotations: basic_block begin_bcip end_bcip jitted_addr
; Instruction annotations:
; Register annotations: def_insn

L0:
    ; PREDS()
    ; BEGIN_BCIP=0x0000 END_BCIP=0x0000

7F7E6AA1A9C8  mov r8, [r15+0xCA0]
7F7E6AA1A9CF  mov r10, [r15+0xC98]
7F7E6AA1A9D6  mov r9, r8
7F7E6AA1A9D9  add r9, 0x240
7F7E6AA1A9E0  cmp r9, r10
7F7E6AA1A9E3  jnbe 0x00007F7E6AA1AAB5
7F7E6AA1A9E9  sub r9, 0x120
7F7E6AA1A9F0  mov [r15+0xCA0], r9
7F7E6AA1A9F7  mov r9, r8
7F7E6AA1A9FA  add r9, 0x5C
7F7E6AA1A9FE  mov [r8+0x30], r9
```